app/utils/navigators/PartnershipNavigator.scala

```diff
@@ -116,4 +124,29 @@ class PartnershipNavigator @Inject()(val dataCacheConnector: UserAnswersCacheCon
       case _ => NavigateTo.dontSave(controllers.routes.SessionExpiredController.onPageLoad())
     }
   }
-}
\ No newline at end of file
+
+  private def regionBasedNameNavigation(answers: UserAnswers): Option[NavigateTo] = {
+    answers.get(AreYouInUKId) match {
+      case Some(false) => NavigateTo.dontSave(routes.PartnershipRegisteredAddressController.onPageLoad())
+      case _ => NavigateTo.dontSave(routes.ConfirmPartnershipDetailsController.onPageLoad())
```

Comment:
case _ could be Some(true) or None - should the behaviour be the same for both cases?

Reply:
fixed